Playmate/View Controllers/Home Tab/Sessions Related/UpcomingSessionsViewController.m

```diff
@@ -14,7 +14,7 @@
 #import "Constants.h"
 #import "Strings.h"
 
-@interface UpcomingSessionsViewController () <UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UICollectionViewDataSourcePrefetching, HomeViewControllerDelegate, SessionCollectionCellDelegate, DZNEmptyDataSetSource, DZNEmptyDataSetDelegate>
+@interface UpcomingSessionsViewController () <UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, HomeViewControllerDelegate, SessionCollectionCellDelegate, DZNEmptyDataSetSource, DZNEmptyDataSetDelegate>
```

Comment:
tiny nit, I'd organize delegate conformance in alphabetical order to make delegates easy to find when reading but it's up to you and not a functional issue